src/lib/list/selection-list.ts

```diff
@@ -392,10 +406,6 @@ export class MatSelectionList extends _MatSelectionListMixinBase implements Focu
 
   /** Passes relevant key presses to our key manager. */
   _keydown(event: KeyboardEvent) {
-    if (this.disabled) {
```

Comment:
This check still has to apply to the `ENTER` and `SPACE` keypresses.

Reply:
Good point.